### PR TITLE
fix: issue #615

### PR DESCRIPTION
**Pull request by yoution**
@maxceem please review;
for testing locally, can you hardcode the id?
hardcode the file `src/services/NotificationsSchedulerService.js`
line 36, hardcode the `projectId`
```
  project = await helper.getProjectById(helper.getAuditM2Muser(), 17232)
```
line 115 and line 116, hardcode the `uuid`
```
  const hostUserDetails = await helper.getUserDetailsByUserUUID('98ec2c16-442e-4b61-8ad1-66123ee37d3c')
  const userDetails = await helper.getUserDetailsByUserUUID('98ec2c16-442e-4b61-8ad1-66123ee37d3c' )
```

### CODE REVIEW

File: src/services/NotificationsSchedulerService.js [hunk 1]
```diff
@@ -44,12 +44,12 @@ async function getProjectWithId (projectId) {
...
 /**
  * extract the members of projects and build recipients list out of them
- * we can use `userId` to identify recipients
+ * we can use `email` to identify recipients
  * @param project the project
  * @returns {string[]} array of recipients
  */
 function buildProjectTeamRecipients (project) {
-  const recipients = _.unionBy(_.map(project.members, m => _.pick(m, 'userId')), 'userId')
+  const recipients = _.unionBy(_.map(project.members, m => _.pick(m, 'email')), 'email')
```
Comment (maxceem) [inline src/services/NotificationsSchedulerService.js:52]: @yoution Emails could be sent by `userId`. I'm not sure that project members would always have `email` addresses. Can we keep `userId` here, or this breaks something?

**Comment by maxceem** [comment]
@yoution I hardcode the lines as you suggested, and run TaaS API from scratch including `npm run local:init` and run the `npm run demo-email-notifications` though I didn't get any emails and I can see some errors in log:

```
[2021-12-23T08:25:00.129Z] InterviewService partiallyUpdateInterviewById ERROR : Cannot read property 'nylasCalendars' of null
[2021-12-23T08:25:00.129Z] NotificationSchedulerService ERROR : sendInterviewExpiredNotifications Service function error: TypeError: Cannot read property 'nylasCalendars' of null
[2021-12-23T08:25:01.865Z] NotificationSchedulerService ERROR : sendInterviewComingUpNotifications Service function error: Error: {"id":"-b3b31b6:17dd750084b:-22d0","result":{"success":true,"status":500,"metadata":null,"content":"Unable to unmarshall the string [{\"streetAddr2\":\"address_2\",\"zip\":\"47579\",\"createdAt\":\"2021-12-22T03:34:58.163Z\",\"streetAddr1\":\"123 Main Street\",\"updatedBy\":\"mess\",\"city\":\"Santa Clause\",\"createdBy\":\"mess\",\"stateCode\":\"IN\",\"type\":\"HOME\",\"updatedAt\":\"2021-12-22T03:34:58.163Z\"}]into interface java.util.List"},"version":"v3"}
[2021-12-23T08:25:02.264Z] NotificationSchedulerService ERROR : sendInterviewCompletedNotifications Service function error: Error: {"id":"-b3b31b6:17dd750084b:-22cf","result":{"success":true,"status":500,"metadata":null,"content":"Unable to unmarshall the string [{\"streetAddr2\":\"address_2\",\"zip\":\"47579\",\"createdAt\":\"2021-12-22T03:34:58.163Z\",\"streetAddr1\":\"123 Main Street\",\"updatedBy\":\"mess\",\"city\":\"Santa Clause\",\"createdBy\":\"mess\",\"stateCode\":\"IN\",\"type\":\"HOME\",\"updatedAt\":\"2021-12-22T03:34:58.163Z\"}]into interface java.util.List"},"version":"v3"}
```

So it doesn't work for me as per local testing.

**Comment by maxceem** [comment]
@yoution I've tested by hardcoding other users and it seems to work good for me localy
```
const hostUserDetails = await helper.getUserDetailsByUserUUID('7669c1d3-f300-466b-8e77-792c3c65ff7d')
const userDetails = await helper.getUserDetailsByUserUUID('5659f247-823e-4979-aebd-d1083209f5bd')
```

I'm merging this PR for QA. But could you please let me know if we can revert the change for Project members above? https://github.com/topcoder-platform/taas-apis/pull/650#pullrequestreview-839114759

**Review by maxceem — COMMENTED** [review]
@yoution merging for QA.

**Comment by yoution** [comment]
@maxceem I will revert to `userid`, wait a moment

**Comment by maxceem** [comment]
@yoution already merged :-), could you please create a new PR with reverted change.

**Comment by yoution** [comment]
@maxceem  https://github.com/topcoder-platform/taas-apis/pull/651

**Comment by maxceem** [comment]
@yoution thanks, merged.

**Pull request merged**